lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -854,7 +856,7 @@ class VerilogEmitterState {
   const GlobalNameTable &globalNames;
 
   /// The stream to emit to.
-  raw_ostream &os;
+  llvm::formatted_raw_ostream &os;
```

Comment:
Yeah this won't, can't be accurate but is close-ish and we can improve it in the future.

100% PP extension is planned (for me anyway)/the way to go, thanks for pointing this out, I have thoughts and happy to guide/help anyone working on this (or take an AI on it myself as needed).  Conceptually I think it's pretty clear re:either adding a "tell me when/where this token gets printed" Token w/opaque info or especially begin/end token (perhaps via side data-structure/listener to keep size under control and not bloat all tokens) that can report a range.  The latter could be built with the former, so shrug.

This PR only somewhat works because the point it queries the stream is right (before!) the newline, which means none of the tokens (probably) have been emitted for this "line" yet (and may be emitted across many).  So it'll be close-ish, I think, but should not be relied upon precisely but gets the right general area.

The errors in these locations should not /compound/ at least.

Reply:
Yeah, that makes sense. I agree that `formatted_raw_ostream` seems working because `getLine/getColumn` is called at the end of the line. PP extension is definitely right way to go.